packages/react-microlink/src/index.js

```diff
@@ -5,8 +5,8 @@ import {CardWrap, CardMedia, CardContent, CardEmptyState} from './components/Car
 import {getUrlPath, someProp, createApiUrl} from './utils'
 
 class Microlink extends Component {
-  constructor (props) {
-    super(props)
```

Comment:
pass `props` is a convention for inherit components

Reply:
Hmm, you're probably right, but what's the usecase? I thought it was only necessary to pass `props` if they were used in the constructor.

Reply:
For example, when you create a component based on `MicrolinkCard`:

```jsx
Class MyCustomComponent extend MicrolinkCard {
 constructor(props) {
   super(props) ← MicrolinkCard constructor
 }
}
```

Reply:
yes you are right, just was a convention, no problem avoiding passing it 👍